### PR TITLE
Documentation grammar changes

### PR DESCRIPTION
**Pull request by uKL**

### CODE REVIEW

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java [hunk 1]
```diff
@@ -49,13 +49,13 @@ public void call(Emitter<T> emitter) {
...
     /**
-     * This method will be overridden in a concrete operation implementations and will contain specific operation logic.
+     * This method should be overridden in a concrete operation implementations and should contain specific operation logic.
      *
      * Implementations should call emitter methods to inform the outside world about emissions of `onNext()`/`onError()`/`onCompleted()`.
      * Implementations must call {@link RadioReleaseInterface#release()} at appropriate point to release the radio for any other operations
      * that are queued.
      *
-     * If the emitter has been canceled it is response of the operation to call {@link RadioReleaseInterface#release()} when possible
+     * If the emitter is cancelled it is a responsibility of the operation to call {@link RadioReleaseInterface#release()}, when possible
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java]: Maybe ` If the emitter is cancelled, a responsibility of the operation is to call` or ` If the emitter is cancelled the operation is obligated to call `?

File: README.md [hunk 2]
```diff
@@ -288,7 +288,7 @@ The below table contains an overview of used `Observable` patterns
 | RxBleConnection | queue() | User defined | User defined |
 | LongWriteOperationBuilder | build() | Single | true |
 
-\* this `Observable` when unsubscribed closes/cleanups internal resources (i.e. finishes scan, closes a connection, disables notifications)<br>
+\* this `Observable` when unsubscribed closes/cleans up internal resources (i.e. finishes scan, closes a connection, disables notifications)<br>
 \** this `Observable` does emit only a single value and finishes in exactly one situation — when Bluetooth Adapter is not available on the device — in this situation there is no point to monitor other states as the adapter does not appear during runtime.
```
Comment (piotrek1543) [inline README.md]: I think that should be there `BluetoothAdapter` instead of `Bluetooth Adapter`.

Comment (piotrek1543) [reply README.md]: Maybe `... BluetoothAdapter is not available on a device. There is no reason to monitor other states as the adapter does not appear during runtime`.

File: rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java [hunk 3]
```diff
@@ -89,7 +89,7 @@ public static void setLogLevel(@RxBleLog.LogLevel int logLevel) {
      * Returns an infinite observable emitting BLE scan results.
      * Scan is automatically started and stopped based on the Observable lifecycle.
      * Scan is started on subscribe and stopped on unsubscribe. You can safely subscribe multiple observers to this observable.
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java]: Maybe `Subscribe starts scan when unsubsribe stops it`

File: rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java [hunk 4]
```diff
@@ -89,7 +89,7 @@ public static void setLogLevel(@RxBleLog.LogLevel int logLevel) {
...
-     * When defining filterServiceUUIDs have in mind that the {@link RxBleScanResult} will be emitted only if _all_ UUIDs will be present
+     * When defining filterServiceUUIDs have in mind that the {@link RxBleScanResult} will be emitted only if _all_ UUIDs are present
      * in the advertisement.
      * <p>
      * The library automatically handles Bluetooth adapter state changes but you are supposed to prompt
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java]: Maybe `Bluetooth adapter's state` or `BluetoothAdapter's state`

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java [hunk 5]
```diff
@@ -24,8 +24,8 @@
 
     /**
      * A function that returns this operation as an Observable.
-     * When the returned observable will be subscribed this operation will be scheduled
-     * to be run on the main thread in future. When appropriate the call to run() will be executed.
+     * When the returned observable is subscribed this operation will be scheduled
```
Comment (piotrek1543) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java]: missing semicolon after `subscribed`?

File: README.md (hunk 2 above)
Comment (uKL) [reply README.md]: +1

File: rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java (hunk 3 above)
Comment (uKL) [reply rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java]: `Scan is started when the Observable is subscribed and stopped when unsubscribed.` I prefer like this.

File: rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java (hunk 4 above)
Comment (uKL) [reply rxandroidble/src/main/java/com/polidea/rxandroidble/RxBleClient.java]: +1

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java (hunk 5 above)
Comment (uKL) [reply rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java]: That's correct, there is a dependency between sentences.

File: README.md [hunk 6]
```diff
@@ -351,10 +351,7 @@ When submitting code, please make every effort to follow existing conventions an
 * Paweł Urban (pawel.urban@polidea.com)
 
 ## Contributors, thank you!
-* Michał Zieliński (michal.zielinski@polidea.com)
-* Fracturedpsyche (https://github.com/fracturedpsyche)
-* Andrea Pregnolato (https://github.com/pregno)
-* Matthieu Vachon (https://github.com/maoueh) - custom operations, yay!
+http://bit.ly/rxandroidble-contributors
```
Comment (dariuszseweryn) [inline README.md]: Maybe
`## [Contributors](https://github.com/Polidea/RxAndroidBle/graphs/contributors), thank you!`?

File: rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java [hunk 7]
```diff
@@ -49,14 +49,15 @@ public void call(Emitter<T> emitter) {
     }
 
     /**
-     * This method will be overridden in a concrete operation implementations and will contain specific operation logic.
+     * This method should be overridden in a concrete operation implementations and should contain specific operation logic.
```
Comment (dariuszseweryn) [inline rxandroidble/src/main/java/com/polidea/rxandroidble/internal/RxBleRadioOperation.java]: must?

**Pull request merged**